api/src/test/java/edu/wpi/first/shuffleboard/api/sources/recording/SerializationTest.java

```diff
@@ -22,6 +22,10 @@
       0, 0, 0, 3, 'b', 'a', 'r'  // "bar", encoded with length
   };
 
+  // Grinning emoji, four bytes
+  @SuppressWarnings("AvoidEscapedUnicodeCharacters")
+  private static final String grinningEmoji = "\uD83D\uDE01";
```

Comment:
Doesn't java support emojis in code now?  Can we put the emoji here instead of the escape sequence?

Reply:
Huh, I didn't know that. That'll make it easier